integration-tests/test/neighborhood.js

```diff
@@ -1,68 +1,54 @@
 import {neighborhood} from "./lib/sources.js";
 import {NEIGHBORHOOD} from "./lib/MiniAppType.js";
-import {verifyMessagesReceived} from "./helpers/testHelpers.js";
+import {
+  INITIAL_STATUS_MESSAGES,
+  EXIT_STATUS_MESSAGE,
+  assertMessagesEqual,
+  verifyMessagesReceived
+} from "./helpers/testHelpers.js";
 import {expect} from "chai";
 
 describe("Neighborhood", () => {
   it("Paints single square", (done) => {
     const expectedMessages = [
-      {type: "STATUS", value: "COMPILING"},
-      {type: "STATUS", value: "COMPILATION_SUCCESSFUL"},
-      {type: "STATUS", value: "RUNNING"},
+      ...INITIAL_STATUS_MESSAGES,
       {
         type: "NEIGHBORHOOD",
         value: "INITIALIZE_PAINTER",
         detail: {
           x: "0",
           y: "0",
           paint: "1",
-          direction: "east"
+          direction: "east",
+          id: "painter-1"
```

Comment:
The `id` fields here are not strictly necessary for this test, but I thought it was clearer to show the expected structure of the detail object even if we don't use it.